templates/security/disclosure-policy.html

```diff
@@ -0,0 +1,124 @@
+      </p>
+      <p>
+        The Ubuntu distribution is divided into multiple pockets: main, universe, restricted, and multiverse. Packages in main are supported by the Ubuntu Security Team. Packages in universe and multiverse are supported by the community; the Ubuntu Security Team can sponsor fixes prepared and tested by community members.
+      </p>
+      <p>
+        Packages in restricted are supported by Canonical's business partners. The Ubuntu Security Team can coordinate with our partners.
+      </p>
+      <p>
+        Software written by Canonical, but delivered outside of Ubuntu, is supported by different teams at Canonical. The Ubuntu Security Team is happy to coordinate communication between external entities (i.e. analysts, reporters) and supporting teams within Canonical, as well as provide guidance and feedback.
+      </p>
+      <p>
+        The Canonical Launchpad code hosting service, Canonical Snap Store, and Canonical Juju Charm Store allows anyone to publish software to users. Launchpad, the Snap Store, and the Juju Charm Store provide a way to contact publishers. As per the <a href="/legal/developer-terms-and-conditions">terms and conditions for these services</a>, publishers are solely responsible for support of their software. If you believe any of these services are being used to host or distribute malicious software, this can be reported either to the Ubuntu Security team or to the relevant platform as appropriate.
+      </p>
+      <p>
+        Ubuntu and Canonical software is distributed through many channels: Canonical-operated download sites, public cloud providers, and community-operated mirrors. Sometimes security issues may be due to customizations at specific providers or distributors; in which case we may ask reporters to contact another party for support.
+      </p>
+      <h2>
+        Out of scope
+      </h2>
+      <p>
+        We will not issue CVEs or fixes for software that is no longer supported. Please check if found issues affect supported versions of software.
+      </p>
+      <p>
+        Not all bugs are vulnerabilities. We use a common understanding of Internet-connected multi-user computers where some of the user accounts may have privileges. Because of this, our idea of what constitutes a vulnerability may not match definitions used by other organizations. We cannot promise every issue reported to us as a security vulnerability will be handled as one; when we differ, we will endeavor to explain our reasoning.
+      </p>
+      <h2>
+        What to expect from us
+      </h2>
+      <p>
+        We intend to provide an initial response to reporters within two business days.
+      </p>
+      <p>
+        The Ubuntu Security Team can assign CVE numbers for issues in Canonical software, as well as anything shipped in Ubuntu. We may direct the reporter to use https://cveform.mitre.org for publicly known issues in non-Canonical software to avoid duplicate assignments.
```

Comment:
this still needs to be a link
```suggestion
        The Ubuntu Security Team can assign CVE numbers for issues in Canonical software, as well as anything shipped in Ubuntu. We may direct the reporter to use <a class="p-link--external' href="https://cveform.mitre.org">cveform.mitre.org</a> for publicly known issues in non-Canonical software to avoid duplicate assignments.
```